### PR TITLE
MO overflow android

### PR DESCRIPTION
**Pull request by yassinecc**

### CODE REVIEW

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/mo-overflow-android/react-native/react/enable-overflow-android.mo.md">enable-overflow-android.mo.md</a>**: Seems you do not need a *Troubleshoot* part
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: README.MD [hunk 1]
```diff
@@ -67,7 +67,7 @@ we help him solve his problem and then together write either a [Standard] or a [
 
 ### React
 - [[Standard] Binding functions in A React component](/react-native/react/binding-functions-in-react-component.s.md)
-- [[MO] Trigger an action on props update](/react-native/react/trigger-action-on-props-update.mo.md)
+- [[MO] Enable component overflow on Android](/react-native/react/enable-overflow-android.mo.md)
```
Comment (tychota) [inline README.MD]: Nope. Pas de replacement

File: react-native/react/enable-overflow-android.mo.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+## Motivation
+
+It is usual to position React components in such a way that every child is rendered on top of its parent. Sometimes you want to do something out of the ordinary and let one component overflow on another that is outside its parent. It turns out that Android is quite picky with this and will not render the overflowing part on MapViews. But fear not, there is a way to get around this!
+
+## Prerequisites
+
+Our setup will include one child view that we want rendered on top of its MapView neighbour:
+
+![Without uncle](../../assets/withoutUncle.png)
```
Comment (tychota) [inline react-native/react/enable-overflow-android.mo.md]: image passe pas.

File: react-native/react/enable-overflow-android.mo.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+- The trick here is to introduce `uncle`, a fourth component which will host the child outside the parent. The uncle will have a negative `marginTop` value so that it overlaps with the previously rendered `neighbour` component:
+
+```jsx
+const OVERLAP_HEIGHT = 50
+
+const styles = {
+  uncle: {
+    backgroundColor: 'transparent',
+    marginTop: - OVERLAP_HEIGHT
```
Comment (tychota) [inline react-native/react/enable-overflow-android.mo.md]: -O pas - O

File: react-native/react/trigger-action-on-props-update.m.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
+# Trigger action on props update with componentWillReceiveProps
```
Comment (tychota) [inline react-native/react/trigger-action-on-props-update.m.md]: Pourquoi y'a 2 standards en 1 ?

**Review by tychota — CHANGES_REQUESTED** [review]

**Comment by louiszawadzki** [comment]
@yassinecc c'est mergé ;)

**Pull request merged**